composer.json

```diff
@@ -47,7 +47,7 @@
         "amphp/php-cs-fixer-config": "dev-master",
         "league/uri-components": "^1.7",
         "monolog/monolog": "^1.23",
-        "phpunit/phpunit": "^6",
+        "phpunit/phpunit": "^6,<8",
```

Comment:
`^6` does already forbid installing `8`, it's equivalent to `>=6 && < 7`.

Reply:
I thought so as well, but composer somehow decided to provide me with PHPunit 8 in the v1.x branch.

Reply:
Strange, maybe a composer.lock from master spilled over. After cleaning everything, PHPunit was correctly pulled. I'll update the PR accordingly.